### PR TITLE
Improve completion display by mapping more symbol kinds

### PR DESCRIPTION
**Pull request by DustinCampbell**

### CODE REVIEW

File: src/features/completionItemProvider.ts [hunk 1]
```diff
@@ -79,14 +79,27 @@ export default class OmniSharpCompletionItemProvider extends AbstractSupport imp
...
+_kinds['Struct'] = CompletionItemKind.Class; // need a better option for this.
+
+// variables
+_kinds['Local'] = CompletionItemKind.Variable;
+_kinds['Parameter'] = CompletionItemKind.Variable;
+_kinds['RangeVariable'] = CompletionItemKind.Variable;
+
+// members
+_kinds['EnumMember'] = CompletionItemKind.Property; // need a better option for this.
+_kinds['Event'] = CompletionItemKind.Field; // need a better option for this.
```
Comment (filipw) [inline src/features/completionItemProvider.ts]: I believe there is a long standing issue to add an event icon to that completion SVG in VS Code repo. Quite a shame it's still not there

Comment (DustinCampbell) [inline src/features/completionItemProvider.ts]: Yes, I piled on https://github.com/Microsoft/vscode/issues/2628 a little while ago.

Comment (filipw) [inline src/features/completionItemProvider.ts]: aha - that one!

**Comment by filipw** [comment]
on a related note.. I didn't realize this is already there https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Symbols/SymbolKind.cs#L108 😄

**Comment by DustinCampbell** [comment]
Well, yeah! We gotta keep moving on the language. :smile:

**Pull request merged**